services/subscriber/udp.go

```diff
@@ -0,0 +1,40 @@
+package subscriber
+
+import (
+	"net"
+
+	"github.com/influxdb/influxdb/cluster"
+)
+
+// Writes points over UDP using the line protocol
+type UDP struct {
+	addr string
+}
+
+func NewUDP(addr string) *UDP {
+	return &UDP{addr: addr}
+}
+
+func (u *UDP) WritePoints(p *cluster.WritePointsRequest) (err error) {
+	var addr *net.UDPAddr
+	var con *net.UDPConn
+	addr, err = net.ResolveUDPAddr("udp", u.addr)
```

Comment:
Not sure if `UDP` instances are long-lived or not, but resolving the addr and creating the `con` below could probably be done once at some point.  Not a big deal though.